docs/kubernetes/KubernetesPolicy.md

```diff
@@ -0,0 +1,62 @@
+# Programming Calico Policy in Kubernetes
+The [Calico-Kubernetes plugin](https://github.com/projectcalico/calico-docker/blob/master/docs/kubernetes/KubernetesIntegration.md) will allow you to program Networking policy directly from your Kubernetes pod config. 
+
+
+### The Basics
+The Calico Networking plugin will look for a `policy` key in the annotations section of your pod's metadata. This `policy` key should map to a single string that outlines networking behavior semantically.
+
+Programming policy is as simple as saying what you want your pod to do!
+
+For instance:
+```
+...
+  metadata:
+    annotations:
+      policy: "allow tcp from label name=backend"
+...
+```
+
+To specify multiple rules, separate them by semicolons.
+```
+...
+      policy: "allow from label name=backend; allow tcp to ports 4001,443"
+...
+```
+In it's current state, Calico supports whitelist oriented, inbound rules. This means that anything unspecified by the policy spec will be rejected at the destination. 
```

Comment:
Calico supports whitelist and blacklist oriented inbound and outbound rules.

The Calico Kubernetes Plugin supports whitelist oriented inbound rules.

"This means that any traffic not specified in a pod's policy is unauthorized, and unauthorized traffic will be dropped at the receiving Kubernetes node."